client/src/main/resources/templates/css/htmlreporter.css

```diff
@@ -263,7 +265,8 @@ h2.yuk_grey_midpnl_ltitle {padding:7px 9px 3px 9px; margin:0; color:#5d5d5d; fon
     width: 99%;
 }
 
-h2.yuk_grey_midpnl_ltitle {width =95%;
+h2.yuk_grey_midpnl_ltitle {
+    width: 95%;
     padding: 7px 9px 3px 9px;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Using width with padding can sometimes make elements larger than you expect.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1249607196/issues/source?bid=2928425&fileBranchId=2937920#l270)